tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
`tsm`, `typescript` remove, too?

Reply:
Yes, let’s remove them (and from running unit tests).

I use `tsm` to write tests in TS and check TS types. But I think in 90% probability our projects will not need TS in tests, because we will emulate configs and input Markdowns (very likely that our tests will be more like integration testing).